uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -640,83 +727,27 @@ define([
       });
     }
 
-    /**
-     * This should be called whenever a date has been changed
-     * First it syncs `from` and `to` date, if it's in 'single day' mode
-     * Then, if all the dates are there, it gets the balance change
-     *
-     * @param {Date} date - the selected date
-     * @param {String} dayType - set to from if from date is selected else to
-     * @return {Promise}
-     */
-    function updateAbsencePeriodDatesTypes (date, dayType) {
-      return vm.loadAbsencePeriodDatesTypes(date, dayType)
-        .then(function () {
-          return vm.updateBalance();
-        })
-        .catch(function (errors) {
-          handleError(errors);
-          vm._setDateAndTypes();
-          vm._toggleBalance();
-        });
-    }
-
     /**
      * Whenever the absence type changes, update the balance opening.
      * Also the balance change needs to be recalculated, if the `from` and `to`
      * dates have been already selected
      */
-    function updateBalance () {
-      vm.selectedAbsenceType = getSelectedAbsenceType();
+    function updateOpeningBalance () {
```

Comment:
why not `setOpeningBalance`? We have plenty of `setXYZ` methods, but only this one `update` method. Let's keep things consistent

Reply:
Done.